bin/hbase

```diff
@@ -658,7 +663,10 @@ elif [ "$COMMAND" = "mapredcp" ] ; then
     for f in "${HBASE_HOME}"/lib/client-facing-thirdparty/*.jar; do
       if [[ ! "${f}" =~ ^.*/htrace-core-3.*\.jar$ ]] && \
          [ "${f}" != "htrace-core.jar$" ] && \
-         [[ ! "${f}" =~ ^.*/slf4j-log4j.*$ ]]; then
+         [[ ! "${f}" =~ ^.*/log4j.*$ ]] && \
+         [[ ! "${f}" =~ ^.*/slf4j.*$ ]] && \
+         [[ ! "${f}" =~ ^.*/jcl-over-slf4j.*$ ]] && \
+         [[ ! "${f}" =~ ^.*/jul-to-slf4j.*$ ]]; then
```

Comment:
Make a function of this filtering so don't have to repeat code?

Reply:
Let me check this.